### DISCUSSION TITLE
Can't login in Step 7 of tutorial

### DISCUSSION THREAD

**Discussion by firstlove75** [prisma/prisma#19975] [Q&A] [answered]
I'm following this tutorial and I get stuck in Step 7. After clicking **Signin with Github** button, I got error **ERR_CONNECTION_CLOSED** on browser of my local environment. Could you help me? Thank you.
https://vercel.com/guides/nextjs-prisma-postgres#step-7.-add-log-in-functionality

**Comment by nurul3101** [comment, marked as answer]
Hello @firstlove75 👋 

Here's a related GitHub Discussion that discusses the same issue that you are getting. Can you have a look at the solution suggested?
- https://github.com/prisma/prisma/discussions/19276
